Mathlib/AlgebraicTopology/SimplicialSet/Rev.lean

```diff
@@ -0,0 +1,60 @@
+/-
+Copyright (c) 2025 Joël Riou. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Joël Riou
+-/
+import Mathlib.AlgebraicTopology.SimplicialObject.Rev
+import Mathlib.AlgebraicTopology.SimplicialSet.Basic
+
+/-!
+# The covariant involution of the category of simplicial sets
+
+In this file, we define the covariant involution of the category
+of simplicial sets that is induced by the
+covariant involution `SimplexCategory.rev : SimplexCategory ⥤ SimplexCategory`.
+
+## TODO (@joelriou)
+
+Show that this involution sends `Δ[n]` to itself, and that via
+this identification, the horn `horn n i` is sent to `horn n i.rev`.
+
+-/
+
+universe u
+
+open CategoryTheory
+
+namespace SSet
+
+/-- The covariant involution of the category of simplicial sets that
+is induced by `SimplexCategory.rev : SimplexCategory ⥤ SimplexCategory`. -/
+@[simps! map]
+def revFunctor : SSet.{u} ⥤ SSet.{u} := SimplicialObject.revFunctor
```

Comment:
I would argue that this should be called `opFunctor` (and the abbrev `op`), and not `revFunctor`: in the case of a nerve of a category, this sends `nerve C` to `nerve Cᵒᵖ`, and when working with quasicategories this involution is the standard way of defining the opposite quasicategory.
I am specifically worried that calling it `revFunctor` and `rev` here would coin a mathlibism and ultimately make quasicategory theory akward by having `rev`s in statements where most users would expect to read `op`.